jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java

```diff
@@ -1,5 +1,6 @@
 package io.github.epam.vuetify.tests.common;
```

Comment:
Maybe it's easier to create new PR and delete this one? It fails on compilation every time :( This approach helped Anastasia with failing on compilation PR.